rust/lance-file/src/v2/writer.rs

```diff
@@ -137,6 +137,13 @@ impl FileWriter {
         }
     }
 
+    /// Returns the format version that will be used when writing the file
+    pub fn version(&self) -> LanceFileVersion {
+        self.options
+            .format_version
+            .unwrap_or(LanceFileVersion::Stable)
```

Comment:
For consistency we should probably do `.unwrap_or(LanceFileVersion::default_v2())` which is what we're using in `initialize`.  Although it's pretty much the same thing.

Reply:
Fixed